src/c++/library/CMakeLists.txt

```diff
@@ -275,6 +277,10 @@ if(TRITON_ENABLE_CC_HTTP OR TRITON_ENABLE_PERF_ANALYZER)
       http-client-library EXCLUDE_FROM_ALL OBJECT
       ${REQUEST_SRCS} ${REQUEST_HDRS}
   )
+  add_dependencies(
+    http-client-library
+    proto-library
```

Comment:
Isn't proto-library used for protobuf<->grpc? Why is it needed for HTTP client?

Reply:
Are there any concerns with introducing the new protobuf dependency to the HTTP client, or any alternatives? CC @GuanLuo @tanmayv25